pkgs/development/compilers/llvm/git/clang/gnu-install-dirs.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/cmake/modules/AddClang.cmake b/cmake/modules/AddClang.cmake
```

Comment:
what does this patch do?

Reply:
Installs some of clang's components right instead of how they do it upstream.